releases.md

```diff
@@ -40,6 +40,13 @@ Further documentation available:
 
 ## Releases
 
+### v0.22
+
+- **Latest Release**: [v0.22.1][v0-22-1] (2023-01-13) ([docs][v0-22-1-docs], [examples][v0-22-1-examples])
+- **Initial Release**: [v0.21.0][v0-22-0] (2022-11-16)
+- **End of Life**: 2023-03-15
+- **Patch Releases**: [v0.22.1][v0-22-1] [v0.21.0][v0-22-0]
+
 ### v0.21
 
 - **Latest Release**: [v0.21.0][v0-21-0] (2022-09-06) ([docs][v0-21-0-docs], [examples][v0-21-0-examples])
```

Comment:
We need to update this and remove v0.21.0 :thinking:

Reply:
🤦

Reply:
Actually I'm not sure - I think the latest release is per released version? See https://github.com/tektoncd/pipeline/blob/main/releases.md And I don't think we had a patch release for v0.21 so 0.21.0 makes sense as the latest release for v0.21

Reply:
as per the Pipeline reference 

```
v0.22
Latest Release: [v0.22.1](https://github.com/tektoncd/triggers/releases/tag/v0.22.1) (2023-01-13) ([docs](https://github.com/tektoncd/triggers/tree/v0.22.1/docs#tekton-triggers), [examples](https://github.com/tektoncd/triggers/tree/v0.22.1/examples#examples))
Initial Release: [v0.22.0](https://github.com/tektoncd/triggers/releases/tag/v0.22.0) (2022-11-16)
End of Life: 2023-03-15
Patch Releases: [v0.22.1](https://github.com/tektoncd/triggers/releases/tag/v0.22.1) [v0.21.0](https://github.com/tektoncd/triggers/releases/tag/v0.21.0)
```
Initial Release for v0.22.1 would be v0.22.0 :thinking:

Reply:
Fixed! PTAL @savitaashture